### ISSUE TITLE
CSRF on the register page

### ISSUE THREAD

**Issue by weaktyper** [cdevroe/unmark#195]
I'm getting the following error on the register page when i set `csrf_protection` to `TRUE`
```
POST https://<<host>>/register/user 500 (Internal Server Error)
```

**Comment by cdevroe** [comment]
@weaktyper Any idea why this would be or how we can fix it? Feel free to submit a PR.

**Comment by weaktyper** [comment]
Hi @cdevroe .... I really don't know. I'm not a PHP developer so it would take me more time to develop on unmark. I have been using unmark every day since I installed it 2 months ago - it is exactly as I was looking for.

I know I opened a few issues here without contributing - unfortunately, due to not knowing PHP and also due to being quite busy (for the next year at least), I won't be able to contribute.

I thought they might help at least as feedback to enhance the app but by all means feel free to close those issues - as I am not a contributor.

One thing I think I would be able to contribute if you are interested is with adding some sort of docker setup - currently I am running it on docker after quite a bit of wrangling.. it's probably not a perfect setup but it might help people quickly get up and running. I'll open a separate issue for the docker thing if it interests you?
